app/spec/spec_helper.rb

```diff
@@ -1,10 +1,11 @@
 ENV['RACK_ENV'] = 'test'
 
-require File.join(File.dirname(__FILE__), '..', 'app/controller.rb')
+require File.join(File.dirname(__FILE__), '..', 'controller.rb')
 
 require 'capybara'
 require 'capybara/rspec'
 require 'rspec'
+require 'database_cleaner'
```

Comment:
Just to avoid merging conflicts, please include ` require data_mapper` just before `require database_cleaner` thaaaanks